src/ui/style/mixins.scss

```diff
@@ -1,4 +1,5 @@
 @import '@/ui/style/theme.scss';
+@import '@/ui/style/fonts.scss';
```

Comment:
`fonts.scss` is actually imported in a large part of SCSS files of the project. The goal here is to avoid to add it's import in each file. Is there a more appropriate solution or should we close this PR? Thanks